lib/vanagon/platform/windows.rb

```diff
@@ -0,0 +1,203 @@
+class Vanagon
+  class Platform
+    class Windows < Vanagon::Platform
+      # The specific bits used to generate a windows package for a given project
+      #
+      # @param project [Vanagon::Project] project to build a windows package of
+      # @return [Array] list of commands required to build a windows package for the given project from a tarball
+      def generate_package(project)
+        # If nothing is passed in as platform type, default to building a nuget package
+        # We should default to building an MSI once that code has been implimented
+        if project.platform.package_type.nil? || project.platform.package_type.empty?
+          return generate_nuget_package(project)
+        else
+          case project.platform.package_type
```

Comment:
As this is a case of one it probably makes more sense to be rolled into the outer if as an elsif.